tests/CSSList/AtRuleBlockListTest.php

```diff
@@ -43,7 +44,8 @@ public static function provideSyntacticlyCorrectAtRule()
                         display: flex;
                     }
                 }
-            '],
+            ',
+            ],
```

Comment:
The leading and trailing whitespace is arbitrary, and for readability.  So the string content and closing quote should now be futher indented.

Reply:
Done.